sections/fundraising_events_featured_projects_list.md

```diff
@@ -2,7 +2,7 @@
 # Fundraising Event Featured Projects List
 
 ```Cirru
-GET https://api.betterplace.org/de/api_v4/fundraising_events/19267/featured_projects.json
+GET https://api.betterplace.org/de/api_v4/fundraising_events/1/featured_projects.json
```

Comment:
Due to using seeded data locally to generate the docs this now returns 404 record not found. So it probably isn't a good example. Same goes for almost all the other example links. People who consume the API are probably more interested in working examples. Is there a plan to handle this?